GetSubscriptionsFeed/handler.ts

```diff
@@ -143,11 +157,20 @@ export function GetSubscriptionsFeedHandler(
     });
 
     const unsubscriptions = new Array<FiscalCodeHash>();
+
+    profileUnsubscriptionsSet.forEach(pu =>
+      // add all users that deleted its own account
+      unsubscriptions.push(pu)
+    );
+
     serviceUnsubscriptionsSet.forEach(su => {
-      if (!profileSubscriptionsSet.has(su)) {
+      if (
+        !profileSubscriptionsSet.has(su) &&
+        !profileUnsubscriptionsSet.has(su)
+      ) {
         // add all users that unsubscribed from this service, skipping those
         // that created the profile on the same day as the service will not
-        // yet know they exist
+        // yet know they exist or deleted its account
```

Comment:
```suggestion
        // yet know they exist or deleted their account
```

Reply:
Fixed in [8857930](https://github.com/pagopa/io-functions-services/pull/86/commits/885793016f5a3b97cae33ed45819ab596d870f67)